ui/app/AppLayouts/Communities/popups/TransferOwnershipPopup.qml

```diff
@@ -109,9 +110,10 @@ StatusDialog {
 
                 onClicked: {
                     // Pre-populated dialog with the relevant Owner token info:
-                    root.sendModalPopup.preSelectedSendType = Constants.SendType.Transfer
+                    root.sendModalPopup.preSelectedSendType = Constants.SendType.ERC721Transfer
                     root.sendModalPopup.preSelectedAccount = ModelUtils.getByKey(root.accounts, "address", token.accountAddress)
-                    root.sendModalPopup.preSelectedHoldingID = token.key
+                    const uid = token.chainId+"+"+token.tokenAddress.toLowerCase()+"+"+"0" // TODO use getUidForData
```

Comment:
Is this TODO still needed? Is there already a task to address it?

Reply:
Maybe you can create a follow up task for that, so we can take a look at it separately.